### PR TITLE
Added autosklearn model and notebook

### PR DESCRIPTION
**Pull request by Stefan415849**

### CODE REVIEW

**Review by rawan-a-d — CHANGES_REQUESTED** [review]
See comments

**Comment by Stefan415849** [comment]
<p>The numerical and categorical columns are divided for the purpose of training.</p>

---

 <a href='https://app.reviewnb.com/rawan-a-d/Air-Quality-Prediction/pull/18/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Review by Ranim1996 — APPROVED** [review]

**Review by rawan-a-d — APPROVED** [review]

**Review by rawan-a-d — CHANGES_REQUESTED** [review]
There are comments in notebook

**Review by Ranim1996 — CHANGES_REQUESTED** [review]
Check the notebook.ipynb there is an error, fix it :)

**Comment by jarnovdh15** [comment]
<p>fixed</p>

---

 <a href='https://app.reviewnb.com/rawan-a-d/Air-Quality-Prediction/pull/18/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by jarnovdh15** [comment]
<p>fixed</p>

---

 <a href='https://app.reviewnb.com/rawan-a-d/Air-Quality-Prediction/pull/18/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by jarnovdh15** [comment]
<p>fixed</p>

---

 <a href='https://app.reviewnb.com/rawan-a-d/Air-Quality-Prediction/pull/18/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by rawan-a-d** [comment]
<p>I still see it</p>

---

 <a href='https://app.reviewnb.com/rawan-a-d/Air-Quality-Prediction/pull/18/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Review by jarnovdh15 — APPROVED** [review]

File: notebook.ipynb [hunk 1]
```diff
@@ -109,7 +109,7 @@
     "from datetime import datetime, date, timedelta\n",
```
Comment (Ranim1996) [inline notebook.ipynb:109]: <p>There is an error here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rawan-a-d/Air-Quality-Prediction/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='304'/>

**Pull request closed without merging**